### PR TITLE
v2ray: update v2ray-core to 4.23.2

### PR DESCRIPTION
**Pull request by SharerMax**
update v2ray-core to 4.23.2

### CODE REVIEW

File: package/lean/v2ray/Makefile [hunk 1]
```diff
@@ -9,13 +9,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=v2ray
-PKG_VERSION:=4.23.1
+PKG_VERSION:=4.23.2
 PKG_RELEASE:=1
```
Comment (BROBIRD) [inline package/lean/v2ray/Makefile:13]: You'd better change `PKG_RELEASE` to `2`

**Comment by BROBIRD** [comment]
https://github.com/v2ray/v2ray-core/issues/2509

**Comment by liwenjie119** [comment]
看到了一条严重问题的issues： https://github.com/v2ray/v2ray-core/issues/2523
v2ray紧急更新到了4.23.3，后续还要继续修复

**Comment by SharerMax** [comment]
> 看到了一条严重问题的issues： [v2ray/v2ray-core#2523](https://github.com/v2ray/v2ray-core/issues/2523)
> v2ray紧急更新到了4.23.3，后续还要继续修复

问题不大，只要不是vmess裸奔就好

**Pull request merged**